mpisppy/generic_cylinders.py

```diff
@@ -544,9 +552,13 @@ def _proper_bundles(cfg):
         fname = os.path.basename(model_fname)
         sys.path.append(dpath)
         module = importlib.import_module(fname)
-    
+
     cfg = _parse_args(module)
 
+    # Perhaps use an object as the so-called module.
+    if hasattr(module, "get_function_object"):
+        module = module.module_get_function_object(cfg)
```

Comment:
```suggestion
        module = module.get_function_object(cfg)
```
`function` and `object` are very generic words. I still like `get_mpisppy_methods` or something similar.

Reply:
How about get_mpisppy_helper_object?

Reply:
I could live with it :-)

Reply:
done.